messagebus.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2019 AccelByte Inc. All Rights Reserved.
+// This is licensed software from AccelByte Inc, for limitations
+// and restrictions contact your company contract manager.
+
+package messagebus
+
+const (
+	GDPR_DeletionAccount string = "GDPR:deletionAccount"
```

Comment:
what is this used for?

Reply:
this is a topic name, we registered the topic name as a public constant so the client doesn't need to manually set the topic